### DISCUSSION TITLE
Onboarding PR is not created when repo was added to the list

### DISCUSSION THREAD

**Discussion by BEGEMOT9I** [renovatebot/renovate#14406] [Request Help]
Hi! I've tried to connect repo in [Adeo](https://github.com/adeo) and, after adding the repository to the list of connected ones, no onboarding PR was created in it. Could u help with this problem?

**Comment by viceice** [comment]
Check the logs at https://app.renovatebot.com/dashboard

**Comment by BEGEMOT9I** [comment]
Thanks!
